src/runtime/injectStylesIntoStyleTag.js

```diff
@@ -187,7 +187,7 @@ function applyToTag(style, options, obj) {
     style.removeAttribute('media');
   }
 
-  if (sourceMap && btoa) {
+  if (sourceMap && typeof btoa !== 'undefined') {
```

Comment:
`btoa` should be `undefined` for old IE, can you provide screenshot?

Reply:
![image](https://user-images.githubusercontent.com/14938137/90202016-6fd3ff80-de17-11ea-8a8a-aa77da266533.png)

Reply:
'typeof' should be used to check if global variables exist.

Reply:
What is IE version? It is weird, because btoa should be false here

Reply:
In IE9, `btoa` is `undefined`.
Or the following code is also good.
```javascript
if (sourceMap && window.btoa) {
```

Reply:
I am fine with `typeof`, weird, I can't reproduce

Reply:
Please check the console window on this page.

http://okchangwon.xyz/naver/ie9-btoa

![image](https://user-images.githubusercontent.com/14938137/90255391-333bee80-de7f-11ea-8c9e-087d22cbf573.png)

Reply:
error from `ie9-btoa` package, not from `style-loader`